src/ts/greedy-nav/GreedyNav.ts

```diff
@@ -604,20 +604,23 @@ export class GreedyNavMenu {
       navDropdownToggle.addEventListener(
         blurEventName,
         (e: FocusEvent): void => {
-          let lastItem: Nullable<HTMLElement> | undefined = null;
-          const headerLinksInNav: Nullable<HTMLElement> =
-            document.querySelector(
-              `${this.navDropdownSelector} .js-cads-copy-into-nav`
-            );
+          let lastItem: HTMLElement | null | undefined;
+          const headerLinksInNav: HTMLElement | null = document.querySelector(
+            `${this.navDropdownSelector} .js-cads-copy-into-nav`
+          );
 
-          if (headerLinksInNav?.offsetParent !== null) {
```

Comment:
This was the initial reason for the issue - the case in which there were no headerLinksInNav would also fall into this if statement as headerLinksInNav?.offsetParent was returning undefined. So I've added an extra condition to cover the possible scenarios